packages/spelunker-api/src/lib/minimap/tiles.mjs

```diff
@@ -0,0 +1,36 @@
+import mpq from '../mpq/index.mjs';
+import logger from '../utils/logger.mjs';
+
+const log = logger('minimap');
+
+log('generating tile index');
+
+const file = mpq.files.get('textures\\Minimap\\md5translate.trs');
+const source = file.data.toString();
+file.close();
+
+// tile directory -> tile name -> content path
+const tiles = {};
+let tileCount = 0;
+
+for (const line of source.split(/\n|\r\n/)) {
+  if (line.startsWith('dir:') || line.trim().length === 0) {
+    continue;
+  }
+
+  const [tilePath, contentPath] = line.split(/\t+/);
+  const tilePathParts = tilePath.split(/[/\\]/);
+  const tileDirectory = tilePathParts.slice(0, -1).map((t) => t.toLowerCase()).join('/');
+  const tileName = tilePathParts.at(-1).split('.')[0].toLowerCase();
```

Comment:
Not entirely sure but have a feeling we may be able to piggyback on `path.posix` or `path.win32` here to do the heavy lifting of converting between one and the other.

Reply:
I'm trying to recall if `MPQ`s are path delimiter agnostic or not. If not, I'd agree that we could probably use `path.win32.parse` to extract the directory and file name. Regardless, I suspect the `trs` file is always written using `\`, so `path.win32.parse` likely always works.